lib/utils/format.js

```diff
@@ -32,6 +33,21 @@ export function fractionToPercentage(fraction) {
   return Math.round(fraction * 100)
 }
 
+/**
+ * Pretty up a BigNumber using it's corresponding ERC20 decimals value with the proper amount of
+ * trailing decimal precision
+ * @param {*} amountBN BigNumber to format
+ * @param {*} decimals precision to use when converting from BigNumber to string
+ * example: a BigNumber value of 123456 for WBTC which has 8 decimals of precision
+ *          would be formatted as: 0.00123456
+ */
+export const amountWithCommas = (amountBN, decimals) => {
```

Comment:
So I'm definitely not married to the name of this method. `amountWithCommas` isn't good or descriptive, and kind of confusing since we have `numberWithCommas`. Open to suggestions!

Reply:
I feel like we should just merge them. `amountWithCommas(amount, decimals, options)` and check the type of amount `amount` to handle it accordingly (string, number, BigNumber) with `options` letting you override the default `getMinPrecision`